cookbooks/aws-parallelcluster-slurm/files/default/head_node_slurm/slurm/pcluster_fleet_config_generator.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import argparse
+import copy
+import json
+import logging
+
+import yaml
+
+log = logging.getLogger()
+
+
+class CriticalError(Exception):
```

Comment:
Is this exception being used in the patch?

Reply:
I missed it, I was "wrongly" using Exception. Thanks fixed!